docker/launch_spark_history_server_locally.sh

```diff
@@ -1,20 +1,92 @@
-#!/bin/sh
+#!/bin/sh 
+set -e
 # ENTER AWS CREDENTIALS TO RUN THE DOCKER IMAGE LOCALLY
 
-DOCKER_IMAGE="varabonthu/spark-web-ui:1.0.6"
-LOG_DIR="s3a://<bucket_name>/<spark-event-logs-prefix>/"
-ENDPOINT="s3.<region>.amazonaws.com"
+CLASS="org.apache.spark.deploy.history.HistoryServer"
+AWS_REGION="${4:-us-east-1}"
+ENDPOINT="s3.$AWS_REGION.amazonaws.com"
+CONTAINER_NAME="${5:-spark-history-server}"
+do_start(){
+	S3_BUCKET=$1
+	S3_BUCKET_PREFIX=$2
+	LOG_DIR="s3a://$S3_BUCKET/$S3_BUCKET_PREFIX"
+	DOCKER_IMAGE="$USER/spark-web-ui:latest"
```

Comment:
Make this as variable as well. Because $USER has to be a input. `DOCKER_IMAGE="$USER/spark-web-ui:latest"`